src/Puppeteer.ts

```diff
@@ -47,6 +64,9 @@ export class Puppeteer {
   __productName: string;
   _lazyLauncher: ProductLauncher;
```

Comment:
could we try putting `private` on these fields? It might cause TS errors in which case we can leave them for now and I'll tackle them in a later PR...